server/mock/datastore_mock.go

```diff
@@ -1004,10 +1004,10 @@ type BatchInsertVPPAppsFunc func(ctx context.Context, apps []*fleet.VPPApp) erro
 
 type GetAssignedVPPAppsFunc func(ctx context.Context, teamID *uint) (map[string]struct{}, error)
 
-type InsertVPPAppWithTeamFunc func(ctx context.Context, app *fleet.VPPApp, teamID *uint) (*fleet.VPPApp, error)
-
 type SetTeamVPPAppsFunc func(ctx context.Context, teamID *uint, adamIDs []string) error
 
+type InsertVPPAppWithTeamFunc func(ctx context.Context, app *fleet.VPPApp, teamID *uint) (*fleet.VPPApp, error)
```

Comment:
Unrelated, but the generated code was stale (just for the order of functions, nothing critical).